app.py

```diff
@@ -55,14 +55,30 @@ def automate_vehicle():
     return "a"
 
 
-@app.route("/tree")
-def tree():
+@app.route("/init_tree")
+def init_tree():
+    global gt
     gt = GeocodeTree()
-    gt.add_tree([77.71111, 12.96026])
+
+    return "a"
+
+
+@app.route("/add_tree")
+def tree():
+
+    gt.add_tree([77.7111, 12.96026])
     print("hi")
     return "a"
 
 
+@app.route("/stop_tree")
+def stop_tree():
+
+    gt.save_and_return()
+
+    return "a"
+
+
```

Comment:
Can you explain more on these I m not able to get it

Reply:
With stop i meant. stop geocoding trees and save it. anyways i have changed the name to save

Reply:
please wait for detailed description and working. will request a review